src/api/app/controllers/webui/cloud/upload_jobs_controller.rb

```diff
@@ -60,16 +61,6 @@ def validate_configuration_presence
         redirect_to cloud_configuration_index_path unless User.current.cloud_configurations?
       end
 
-      def validate_ec2_configuration_presence
-        redirect_to cloud_ec2_configuration_path if User.current.ec2_configuration.blank?
-      end
-
-      def permitted_params
```

Comment:
Now you need the permitted params again :) Also you need to add the azure parameters as well.